src/components/HorizontalBarChart/index.js

```diff
@@ -16,7 +16,7 @@ const HorizontalBarChart = ({ data, maxValue, title }) => (
 
       return (
         <div className="bar-container" key={key}>
-          <span className="font-frijole bar-chart-position">{index + 1}</span>
+          <span className="font-frijole bar-chart-position">{index === 6 ? 'X' : index + 1}</span>
```

Comment:
```suggestion
          <span className="font-frijole bar-chart-position">{index === MAX_ATTEMPTS ? 'X' : index + 1}</span>
```

here you can use the constant `MAX_ATTEMPTS`